feat: added translations specs (#124)
algolia/instantsearch-specs: content/widgets/geo-search.md

    description: Precision of geo search
  - name: transformItems
    description: Function which receives the items, which will be called before displaying them. Should return a new array with the same shape as the original array. Useful for mapping over the items to transform, remove or reorder them
translations:
  - name: resetButtonText
    default: 'Clear the map refinement'

>     description: The text that will be displayed on the "Clear the map refinement" button at the bottom.

Reviewer: ```suggestion
    description: The text of the "Clear the map refinement" button.
```

  - name: redoButtonText
    default: 'Redo search here'

>     description: A button that will be displayed when the map has been moved.

Reviewer: ```suggestion
    description: The text of the button that's displayed when the map was moved.
```
